src/orion/executor/ray_backend.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import traceback
+
+from orion.executor.base import (
+    AsyncException,
+    AsyncResult,
+    BaseExecutor,
+    ExecutorClosed,
+    Future,
+)
+
+try:
```

Comment:
This should be implemented using the `ImportOptional` module, like [here](https://github.com/Epistimio/orion/blob/develop/src/orion/algo/hebo/hebo_algo.py#L28).